CMakeLists.txt

```diff
@@ -64,6 +64,11 @@ find_package(ECM REQUIRED NO_MODULE)
 set(CMAKE_MODULE_PATH "${ECM_MODULE_PATH};${CMAKE_MODULE_PATH}")
 include(CTest)
 include(ECMAddTests)
+include(ECMQtDeclareLoggingCategory)
+if(UNIX AND NOT APPLE)
+    # only used for kdebugsettings compatibility for logging categories
+    include(KDEInstallDirs)
```

Comment:
this doesn't seem to work with Qt6 yet (maybe when KF6 is out, for now there's only [an alpha version that we shouldn't use](https://api.kde.org/ecm/kde-module/KDEInstallDirs6.html)